### PR TITLE
Make UMD optional, due to https://phabricator.babeljs.io/T7084

### PR DESCRIPTION
**Pull request by jpnelson**

### CODE REVIEW

File: lib/babel.js [hunk 1]
```diff
@@ -4,13 +4,19 @@ var report = require('./report');
...
   opts = opts || {};
+
+  var plugins = resolve(
+    'babel-plugin-add-module-exports',
+    'babel-plugin-syntax-jsx',
+    ['babel-plugin-transform-react-jsx', opts.jsx]
+  );
+
+  if (opts.umd) {
+    plugins.push('babel-plugin-transform-es2015-modules-umd');
```
Comment (joscha) [inline lib/babel.js]: this one doesn't need to be resolved?

Comment (jpnelson) [inline lib/babel.js]: good point

**Comment by joscha** [comment]
LGTM

**Comment by treshugart** [comment]
lgtm

**Comment by jpnelson** [comment]
@treshugart can you merge this? Looks like the lgtm hook is broken

**Pull request merged**